extensions/support-page/renderer.tsx

```diff
@@ -20,9 +18,9 @@ export default class SupportPageRendererExtension extends LensRendererExtension
       item: (
         <div
           className="flex align-center gaps hover-highlight"
-          onClick={() => Navigation.navigate(supportPageURL())}
+          onClick={() => Navigation.navigate(this.getPageUrl(pageUrl))}
```

Comment:
If you instead have a `navigate()` method on `LensRendererExtension` (or `LensExtension`?) then this could be 
```suggestion
          onClick={() => this.navigate(pageUrl)}
```
(the `navigate()` method would call `this.getPageUrl()`, no need to expose it. This aligns with what @jakolehm did in https://github.com/lensapp/lens/pull/1323

Reply:
guess that didn't work out. Would be nice if it were possible ...

Reply:
See #1345

Reply:
Worked in dev/local build :/